openwisp_radius/api/freeradius_views.py

```diff
@@ -431,12 +432,14 @@ def post(self, request, *args, **kwargs):
             acct_data = self._data_to_acct_model(serializer.validated_data.copy())
             serializer.create(acct_data)
             headers = self.get_success_headers(serializer.data)
+            self.send_radius_accounting_signal(acct_data)
             return Response(None, status=201, headers=headers)
         else:
             serializer = self.get_serializer(instance, data=data, partial=False)
             serializer.is_valid(raise_exception=True)
             acct_data = self._data_to_acct_model(serializer.validated_data.copy())
             serializer.update(instance, acct_data)
+            self.send_radius_accounting_signal(acct_data)
```

Comment:
@devkapilbansal Instead of `acct_data`, can we pass `serializer.data`?
Because while passing this data to celery I am getting:
`kombu.exceptions.EncodeError: Object of type Organization is not JSON serializable`

`_data_to_acct_model` removes `status_type` and adds `organization`. Also, `status_type` is more required in detecting the radius status i.e. `Start`.

Reply:
Makes sense.